docs/make.jl

```diff
@@ -162,11 +161,23 @@ makedocs(bib,
 
 @info "Cleaning up temporary .jld2 and .nc output created by doctests or literated examples..."
 
-for file in vcat(glob("docs/*.jld2"),
-                 glob("docs/*.nc"),
-                 glob("docs/build/generated/*.jld2"),
-                 glob("docs/build/generated/*.nc")
-                 )
+"""
+    recursive_find(directory, pattern)
+
+Return list of filepaths within `directory` that fall under the `pattern::Regex`, e.g., `pattern = r"\.jl"`.
+"""
+recursive_find(directory, pattern) =
+    mapreduce(vcat, walkdir(directory)) do (root, dirs, files)
+        joinpath.(root, filter(contains(pattern), files))
+    end
+
+files = []
+
+for pattern in [r"\.jld2", r"\.nc"]
```

Comment:
The error happens with escaping, so maybe the code below will work (works for me locally)

```suggestion
recursive_find(directory, pattern) =
    mapreduce(vcat, walkdir(directory)) do (root, dirs, files)
        joinpath.(root, filter(endswith(pattern), files))
    end

files = []

for pattern in [r".jld2", r".nc"]
```

Reply:
Thanks!!